src/components/Nav/Nav.js

```diff
@@ -1,7 +1,133 @@
 import React from 'react';
+import styled from 'styled-components';
+import { useHistory } from 'react-router';
 
 const Nav = props => {
-  return <div>hi</div>;
+  const history = useHistory();
+
+  return (
+    <NavBox>
+      <NavContainer>
+        <Logo>
+          <i className="fas fa-paint-brush fa-3x"></i>
+          <div className="logoText">
```

Comment:
```suggestion
          <LogoText>
```
가급적 클래스 셀렉터 사용하지 않을 수 있습니다

Reply:
> 가급적 클래스 셀렉터 사용하지 않을 수 있습니다

styled componets 로 만들어서 수정했습니다.